docs/managing-secrets-with-vault.md

```diff
@@ -25,35 +25,219 @@ guide](https://developer.hashicorp.com/vault/tutorials/kubernetes/kubernetes-raf
 _Prerequisites_
 
 * You're logged into the OpenShift Container Platform as a cluster adminstrator.
+* Your OpenShift environment has storage available using Persistent Volume Claims.
 * You've installed `helm` version 3.15 or later.
 
 _Procedure_
 
-* Add the Hashicorp Helm repository:
-  ```bash
-  $ helm repo add hashicorp https://helm.releases.hashicorp.com
-  ```
-* Update all repositories:
-  ```bash
-  $ helm repo update
-  ```
-* Install Hashicorp Vault:
-  * Create a project for Vault:
+1. Add the Hashicorp Helm repository
```

Comment:
We use `*` not numbered bullets.

Reply:
I don't think so, ordered steps must use number for the list. There are examples in RHOSO documentation, as well in the openstack-operator documentations. That's also a feedback I got while working with docs team for the ci-framework.